Gemfile

```diff
@@ -48,6 +53,9 @@ group :development, :test do
   gem 'shoulda-matchers', '~> 3.1.2'
   # Use sqlite3 as the database for Active Record
   gem 'sqlite3'
+  gem 'dotenv-rails', groups: [:development, :test]
+  gem 'activeadmin' , '~> 1.3.1'
+  gem 'pundit', '~> 2.0.0'
```

Comment:
Pundit should be on all workspaces, not only development and testing.

Reply:
done!